<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPOPProp</class>
 <widget class="QWidget" name="IPOPProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>471</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IPOP</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="xmpp_group_box">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>XMPP</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="xmpp_host_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Host:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="xmpp_username_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Username:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="xmpp_password_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Password:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="xmpp_password_layout">
        <item>
         <widget class="KLineEdit" name="xmpp_password">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="passwordMode">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="xmpp_password_type_combo">
          <property name="font">
           <font>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>Always Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Store</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Not Required</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="xmpp_username">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="xmpp_host">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="ip4_group_box">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>IP4</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="KLineEdit" name="ip4_address">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ip4_address_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Address:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="ip4_netmask">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ip4_netmask_label">
        <property name="font">
         <font>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Netmask:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
